include/core/parameters.h

```diff
@@ -1291,42 +1291,99 @@ namespace Parameters
     declare_default_entry(ParameterHandler &prm, unsigned int index);
     void
     parse_parameters(ParameterHandler &prm);
-
-    unsigned int                 nb;
-    unsigned int                 order;
-    unsigned int                 initial_refinement;
-    double                       inside_radius;
-    double                       outside_radius;
-    bool                         time_extrapolation_of_refinement_zone;
+    // Vector of particles
     std::vector<IBParticle<dim>> particles;
-    bool                         calculate_force_ib;
-    bool                         assemble_navier_stokes_inside;
-    std::string                  ib_force_output_file;
-    Tensor<1, dim>               gravity;
-    double                       particle_nonlinear_tol;
-    double                       wall_youngs_modulus;
-    double                       wall_poisson_ratio;
-    double                       wall_rolling_friction_coefficient;
-    double                       wall_friction_coefficient;
-    double                       wall_restitution_coefficient;
-    unsigned int                 coupling_frequency;
-    bool                         enable_lubrication_force;
-    double                       lubrication_range_max;
-    double                       lubrication_range_min;
-    double                       contact_search_radius_factor;
-    int                          contact_search_frequency;
-    bool                         load_particles_from_file;
-    std::string                  particles_file;
-    bool                         enable_extra_sharp_interface_vtu_output_field;
-
-    std::shared_ptr<Functions::ParsedFunction<dim>> f_gravity;
 
-    double      particle_nonlinear_tolerance;
-    double      length_ratio;
-    bool        enable_extrapolation;
-    double      alpha;
-    bool        print_dem;
+    // Number of declared IB particles
+    unsigned int nb;
```

Comment:
Not sure if the variable name is really descriptive...

Reply:
It is not.